src/components/Health/HealthDetails.tsx

```diff
@@ -52,7 +52,7 @@ export class HealthDetails extends React.PureComponent<Props, {}> {
         )}
       </div>
     ) : (
-      <></>
+      <div key={idx}></div>
```

Comment:
If this is just to avoid react warning about keys, iirc you can do instead: `<React.Fragment key={idx}/>` . That way we don't "pollute" the DOM, if i'm correct.

Reply:
I'll see, I didn't know that trick.
Yes, this was to avoid an annoying warning for lack of keys.